source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRImageRenderer.mm

```diff
@@ -45,16 +45,33 @@ - (UIView *)render:(UIView<ACRIContentHoldingView> *)viewGroup
     BOOL isAspectRatioNeeded = !(pixelWidth && pixelHeight);
     CGSize cgsize = [acoConfig getImageSize:imgElem->GetImageSize()];
 
-    NSString *key = [NSString stringWithCString:imgElem->GetUrl().c_str() encoding:[NSString defaultCStringEncoding]];
+    NSString *number = [[NSNumber numberWithUnsignedLongLong:(unsigned long long)(elem.get())] stringValue];
+    NSString *urlString = [NSString stringWithCString:imgElem->GetUrl().c_str() encoding:[NSString defaultCStringEncoding]];
```

Comment:
Ask: Could you try to comment the code a bit to explain what is going on here at all? Thanks!

Reply:
sure

Reply:
sure